src/tests/Dashboard/container/SpecificConcept.test.jsx

```diff
@@ -134,5 +134,28 @@ describe('Dashboard SpecificConcept Component', () => {
     expect(mapStateToProps(initialState).concepts).toEqual([]);
     expect(mapStateToProps(initialState).isFetching).toEqual(false);
   });
+
+  it('should handle change in path', () => {
```

Comment:
Do you mind adding the word "the" before path "should handle change in the path"

Reply:
Thanks, but I think this is right English because if I put the, It might sound like I am talking of one path, but actually, am testing any change in path. We can stick to this I guess.